app/players/elves/druid.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str,

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line